lib/database_consistency/checkers/association_checkers/association_checker.rb

```diff
@@ -23,6 +23,13 @@ def column_or_attribute_name
       def table_or_model_name
         @table_or_model_name ||= model.name.to_s
       end
+
+      def foreign_key_exists?
```

Comment:
This method is calling a rubocop failure even though I copied the contents from an existing method.

Reply:
Feel free to disable it to make Rubocop green.